web/modules/cm-espells/src/MisspellingTooltip.svelte

```diff
@@ -102,9 +102,9 @@
 
   <hr />
 
-  <pre class="code cm-spellcheck-tip-source">
-      <code>{$t("#-source", { slice: word.word, from: word.from, to: word.to })}</code>
-  </pre>
+  <pre class="code cm-spellcheck-tip-source"><code
+      >{$t("#-source", { slice: word.word, from: word.from, to: word.to })}</code
+    ></pre>
```

Comment:
lol, clever workaround to avoid whitespace

Reply:
Yeah, that's what Prettier does. It is a cool workaround.